frontend/packages/ceph-storage-plugin/integration-tests/tests/2-tests/pvc.scenario.ts

```diff
@@ -0,0 +1,86 @@
+import { browser } from 'protractor';
+import { appHost } from '@console/internal-integration-tests/protractor.conf';
+import * as crudView from '@console/internal-integration-tests/views/crud.view';
```

Comment:
Since only `resourceRowsPresent()` is used, maybe it would be enough to only import that method.

Reply:
Done